services/configuration/src/schema.rs

```diff
@@ -54,13 +56,51 @@ lazy_static! {
     ).into_bytes();
 }
 
-/// A functional equivalent to `Option<Vote>` used to store votes in the service schema.
+/// A enum used to represent different kinds of vote, `Vote` and `VoteAgainst` transactions.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "vote_for", rename_all = "lowercase")]
+pub enum VotingDecision {
+    /// `Vote` transaction variant
```

Comment:
Please add a dot at the end of the comment.

Reply:
Done